contracts/StorageMarket_flat.sol

```diff
@@ -1990,18 +2007,25 @@ contract StorageMarketPlace is Sablier {
             _stopTime
         );
         // Once stream is created update both mappings so that the subscription can be tracked easily from both sides
-
-
-            BuyerSubscription[] storage buyerSubscriptions
-         = sellerToBuyer[reciever];
-        buyerSubscriptions.push(
-            BuyerSubscription(msg.sender, _deposit, streamId, true)
-        );
-        SellerSubscription[] storage sellerSubscriptions = buyerToSeller[msg
-            .sender];
-        sellerSubscriptions.push(
-            SellerSubscription(reciever, _deposit, streamId, true)
-        );
+        // Streams storage stream = subscriptions[msg.sender];
+        // stream.streamIds.push(streamId);
+        // subscriptions[msg.sender] = Streams(stream.streamIds);
+        
+        subscriptions[msg.sender].streamIds.push(streamId);
+        subscriptions[reciever].streamIds.push(streamId);
+        emit subscription(streamId);
```

Comment:
if emit the same "subscription" event in all three methods, then we can't query for anything using these events.

so either we go with events with well described names or we just use existing events  from the sabier contract